.envrc.testnet-02

```diff
@@ -1,5 +1,10 @@
 # overrides for testnet-02
-export MIDNIGHT_NODE_IMAGE="midnightnetwork/midnight-node:0.12.0"
+export MIDNIGHT_NODE_IMAGE="midnightnetwork/midnight-node:0.12.1"
+export INDEXER_IMAGE="ghcr.io/midnight-ntwrk/indexer-standalone:2.1.4"
+export INDEXER_NETWORK_ID="TestNet"
```

Comment:
not TestNet-02 ? Honestly I think we should have lowercase for all these network ids.